packages/e2e-test-utils-playwright/src/admin/set-option.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import type { Admin } from './';
+
+export async function setOption( this: Admin, setting: string, value: string ) {
```

Comment:
Personally, I wouldn't put this inside `e2e-test-utils-playwright` since it's just a temporary solution for one test suite. Breaking changes will be made once the REST options API is implemented.

Instead, I would put this inside the `comments` test file to make it clear that it's an internal API.